internal/handlers/common/msg_listcommands.go

```diff
@@ -205,9 +209,18 @@ func MsgListCommands(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
 	cmdList := must.NotFail(types.NewDocument())
 	names := maps.Keys(Commands)
 	sort.Strings(names)
+
+	ignoreLowercaseCmds := map[string]bool{
+		"buildinfo":     true,
```

Comment:
```suggestion
	ignoreLowercaseCmds := map[string]struct{}{
		"buildinfo":     struct{}{},
```

To switch from three possible states (true, false, absent) to two (present, absent)

Reply:
If the command does not exist in the map then the requested key will return false - the zero-value, right? So, there's only ever two states in this map? (true, false|absent).

Reply:
But you listed three states :)
We uses `map[T]struct{}` for sets in FerretDB. Let's use the same data structure there